### PR TITLE
[Storage Manager] persist storage logs 

### PR DESCRIPTION
**Pull request by Binyang2014**

### CODE REVIEW

File: src/storage-manager/deploy/storage-manager.yaml.template [hunk 1]
```diff
@@ -68,6 +68,8 @@ spec:
         - name: host-modules
           mountPath: /lib/modules
```
Comment (abuccts) [inline src/storage-manager/deploy/storage-manager.yaml.template]: what is `/ilb/modules` used for?

File: src/storage-manager/deploy/conf/smb.conf.template [hunk 2]
```diff
@@ -49,7 +49,8 @@
     encrypt passwords = Yes
     restrict anonymous = 2
     log level = 2
-    log file = /var/log/samba/samba.log
+    log file = /var/log/storage/samba.log
+    max log size = 10000
```
Comment (abuccts) [inline src/storage-manager/deploy/conf/smb.conf.template]: 10MB is too small

File: src/storage-manager/deploy/conf/smb.conf.template [hunk 3]
```diff
@@ -49,7 +49,8 @@
     encrypt passwords = Yes
     restrict anonymous = 2
     log level = 2
-    log file = /var/log/samba/samba.log
+    log file = /var/log/storage/samba.log
```
Comment (abuccts) [inline src/storage-manager/deploy/conf/smb.conf.template]:
```suggestion
    log file = /var/log/storage/%m.log
```

File: src/storage-manager/deploy/storage-manager.yaml.template (hunk 1 above)
Comment (Binyang2014) [reply src/storage-manager/deploy/storage-manager.yaml.template]: If think it used for:
https://github.com/microsoft/pai/blob/120179950934692bd304ce5f38a54f0ecba673eb/src/storage-manager/deploy/scripts/entrypoint.sh#L46-L47

File: src/storage-manager/deploy/conf/smb.conf.template (hunk 2 above)
Comment (Binyang2014) [reply src/storage-manager/deploy/conf/smb.conf.template]: Change to 100MB fixed

File: src/storage-manager/deploy/conf/smb.conf.template (hunk 3 above)
Comment (Binyang2014) [reply src/storage-manager/deploy/conf/smb.conf.template]: fixed

File: src/storage-manager/deploy/storage-manager.yaml.template (hunk 1 above)
Comment (Binyang2014) [reply src/storage-manager/deploy/storage-manager.yaml.template]: Refer to https://stackoverflow.com/questions/33013539/docker-loading-kernel-modules

File: src/storage-manager/deploy/conf/smb.conf.template (hunk 2 above)
Comment (abuccts) [reply src/storage-manager/deploy/conf/smb.conf.template]: could you increase to 2GB?

Comment (abuccts) [reply src/storage-manager/deploy/conf/smb.conf.template]: and change `log level` to 3

Comment (Binyang2014) [reply src/storage-manager/deploy/conf/smb.conf.template]: fixed

**Pull request merged**